src/backend/commands/extension.c

```diff
@@ -816,6 +818,12 @@ execute_extension_script(Node *stmt,
 	int			save_nestlevel;
 	StringInfoData pathbuf;
 	ListCell   *lc;
+	bool orig_gp_enable_gpperfmon = gp_enable_gpperfmon;
+
+	if (gp_enable_gpperfmon > DEBUG4)
+	{
+		gp_enable_gpperfmon = false;
```

Comment:
We've decided to disable gpperfmon for create extension nested statements as far as we have only extension  script content as sql string that is non-informative